provided.al2/rust/cookiecutter-aws-sam-hello-rust/{{ cookiecutter.project_slug }}/Makefile

```diff
@@ -1,9 +1,9 @@
 {% if cookiecutter.architecture == 'arm64' -%}
-CARGO_LAMBDA_FLAGS = '--arm64'
+CARGO_LAMBDA_FLAGS = --arm64
 {%- else -%}
-CARGO_LAMBDA_FLAGS = ''
+CARGO_LAMBDA_FLAGS = --target x86_64-unknown-linux-gnu
```

Comment:
Actually this will result in an error, because `make` takes single quotes literallity:
```bash
$ make build
cargo lambda build --release ''
error: Found argument '' which wasn't expected, or isn't valid in this context

USAGE:
    cargo lambda build [OPTIONS]

For more information try --help
Makefile:5: recipe for target 'build' failed
make: *** [build] Error 2
```

Alternatively we can write it as:
```bash
CARGO_LAMBDA_FLAGS = 
```
But it looks look like a typo and someone might try to "fix" it in the future and breaks everything again.
You can find the discussion in #267